iopipe/contrib/trace/plugin.py

```diff
@@ -0,0 +1,45 @@
+from iopipe.plugins import Plugin
```

Comment:
Is the thinking on this to break this out into a separate package later on?

Reply:
@pselle It could, but don't think it needs to be. Python is more "batteries included".

Reply:
@kolanos we can leave it here for now and say we'll discuss product-wise